releasenotes/notes/disable-ironic-prometheus-exporter-894d98022a1e926d.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    The Ironic Prometheus Exporter is now disabled by default. If you need to
+    use the Ironic Prometheus Exporter, you can enable it by setting the
+    `kolla_enable_ironic_prometheus_exporter` option to `true`.
```

Comment:
Might also want to mention that the redfish exporter supersedes it, just to make it obvious that we're not removing functionality